DashKit/Mocks/MockPodCommManager.swift

```diff
@@ -37,10 +37,14 @@ public class MockPodCommManager: PodCommManagerProtocol {
 
     public var podCommState: PodCommState {
         get {
-            return lockedPodCommState.value
+            if let podStatus = podStatus {
```

Comment:
Is `private let lockedPodCommState: Locked<PodCommState>` no longer needed?

Reply:
Also, consider simplifying to just `podStatus?.podCommState ?? .noPod`